### ISSUE TITLE
error nan loss 

### ISSUE THREAD

**Issue by damvantai** [auroua/InsightFace_TF#12]
when i use arcface loss in mnist, my error loss nan ? do you test your code accuracy ?

**Comment by auroua** [comment]
I have not test the code on mnist. I think you can try the following method to make `arcface` work probably on `mnist`. 
1. Try to use small network.
2. The 50-layer network use lots of `batch norm` layer, but the `mnist` dataset only contain 10 category and the variance in each category is not large enough to fit so many `batch norm` layers, so it will cause the `nan`. You should to reduce the number of  `batch norm` layers.

**Comment by damvantai** [comment]
i am using small neural network, but loss error? nan
https://github.com/damvantai/tensorflow-sphereface-asoftmax/blob/master/test_mnist.py

**Comment by damvantai** [comment]
i use sphereface loss is right but arcface loss ? error

**Comment by seanM29** [comment]
i meet same problem on mnist(resnet 50, arcface loss), have you solved this problem?
@damvantai
